mailpoet/assets/js/src/landingpage/content.tsx

```diff
@@ -29,7 +29,10 @@ function Content() {
         <br />
         <br />
         <Heading level={4}>
-          {__('Powering email marketing for 600,000+ websites', 'mailpoet')}
+          {__(
+            'Powering email marketing for [count] websites',
+            'mailpoet',
+          ).replace('[count]', '700,000+')}
```

Comment:
We usually use `sprintf()`, I would also suggest a `// translators` comment before explaining the `%s` and you would need to import `sprintf()`
```suggestion
          {sprintf(__(
            'Powering email marketing for %s websites',
            'mailpoet',
          ), '700,000+')}
```